exercises/practice/isbn-verifier/example.cpp

```diff
@@ -0,0 +1,24 @@
+#include "example.h"
+#include <regex>
+#include <vector>
+
+namespace isbn_verifier {
+    bool is_valid(const std::string& isbn) {
+        auto raw_string = std::regex_replace(isbn, std::regex("-"), "");
+        
+        if(!std::regex_match(raw_string, std::regex("\\d{9}[0-9|xX]"))) {
```

Comment:
`[0-9|xX]` includes the character `'|'`.
Also, I'm curious: Why the lowercase `'x'`?